docs/manuals/source/TasksAndConcepts/Plugins/FileDaemonPlugins/VMwarePlugin.rst.inc

```diff
@@ -490,34 +465,84 @@ After the restore process has finished, the restored VMDK files can be found und
-                ":dc=mydc1:folder=/webservers"
-                ":vmname=websrv1"
-                ":vcserver=vcenter.example.org"
-                ":vcuser=bakadm@vsphere.local"
-                ":vcpass=Bak.Adm-1234"
-                ":vadp_dumper_verbose=yes"
+vcuser (mandatory on backup)
+   Username for API access to vCenter, eg. administrator@vsphere.local
+
+vcpass (mandatory on backup)
+   Password for API access to vCenter
+
+dc (mandatory on backup)
+   Datacenter name
+
+folder (mandatory on backup)
+   The VM folder in which the VM to be backed up resides. This must be given like a UNIX path with ``/`` as separator.
+
+vmname (mandatory on backup)
+   The name of the VM to be backed up.
+
+vcthumbprint (optional)
+   Thumbprint of the vCenter SSL Certificate, which is the SHA1 checksum of the SSL Certificate
+
+transport (optional)
+   Normally the transport mode mode will be autonegotiated, eg. if the system that runs this plugin is a VM has storage access to the datatore of the VM thats beeing backed up, the *hotadd* transport will be used. Otherwise the *nbd* or *nbdssl* transport. For details about transport modes see the VDDK documentation. This option can be used to force the given transport mode.
+
+log_path (optional)
+   The default log path is :file:`/var/log/bareos/`. A different path can be specified using this option, it will be used for ``bareos_vadp_dumper`` log files.
+
+localvmdk (optional)
+   Restore to local ``.vmdk`` file(s) instead of restore to VM. Default is *no*.
+
+vadp_dumper_verbose (optional)
+   When setting ``vadp_dumper_verbose=yes``, the option ``-v`` will be added when running ``bareos_vadp_dumper``. This can be helpful for debugging purposes.
+
+verifyssl (optional)
+   By default the validity of SSL certificates will be verifed. By setting ``verifyssl=no`` this can be disabled.
+
+quiesce (optional)
+   The backed up VM will be triggered to quiesce its filesystems before creating a snapshot by default to increase the data consistency. This can fail or take very long on a VM which runs heavy I/O workload. When setting ``quiesce=no`` the quiescing will be skipped, but the snapshot may be inconsistent. It is not recommended to use this option, instead try stop heavy I/O load before snapshot. This could be possible by running pre-freeze and post-thaw actions which can be configured in VMware tools, see VMware documentation for details.
+
+cleanup_tmp_files (optional)
+   By default, temporary files created by the plugin will be cleaned up after backup or restore. When setting ``cleanup_tmp_files=no`` they will be left over, this can be helpful for debugging purposes. Since :sinceVersion:`22.0.0: VMware Plugin`
+
+restore_esxhost (optional)
+   By default, if a VM to be restored does not exist, it will be recreated on the same host that it has been running on at backup time. Use this option to restore on a the given ESX host. Since :sinceVersion:`22.0.0: VMware Plugin`
```

Comment:
```suggestion
   By default, if a VM to be restored does not exist, it will be recreated on the same host that it has been running on at backup time. Use this option to restore on the given ESX host. Since :sinceVersion:`22.0.0: VMware Plugin`
```